components/Controls.tsx

```diff
@@ -190,3 +221,25 @@ function SetPowerLevelButton({ level, selected, onSelect }: SetPowerLevelButtonA
     )
 }
 
+interface SetBellToneButtonArgs {
+    tone: string
+    selected: boolean
+    onSelect(): void
+}
+
+function SetBellToneButton({ tone, selected, onSelect }: SetBellToneButtonArgs) {
+    return (
+        <Button
+            onClick={onSelect}
+            style={{
+                margin: 4,
+                padding: '6px 10px',
+                width: 50,
+                height: 50,
+                backgroundColor: selected ? 'var(--active-button-bg-color)' : undefined,
+            }}
+        >
+            <h1 style={{ margin: 0 }}>{tone}</h1>
```

Comment:
I think we should add a description below this like the speed limit buttons

![image](https://user-images.githubusercontent.com/15320763/179351689-7ca264cc-a348-4622-b0d4-f1833e058680.png)

Reply:
Which descriptions should I add?

Boat, bell & party?

Reply:
Sounds good!

Reply:
I actually might have found the submarine  horn, when I get back home I will check if it works and I will implement it. 

Bell: 16
Submarine: 0a
Party: 17
Foghorn: 18